tests/fixtures/input-plugins/input-plugin-checkbox.js

```diff
@@ -33,5 +33,5 @@ module.exports = {
       },
     },
   },
-  html: `<label for="{{checkboxSelector.id}}">{{checkboxSelector.label}}</label>{% for option in checkboxSelector.options %}<label><input type="{{checkboxSelector.type}}" name={{checkboxSelector.name}} id="{{checkboxSelector.id}}" value="{{checkboxSelector.value}}" {% if option.value == checkboxSelector.value %}checked{% endif %}>{{option.label}}</label>{% endfor %}`,
-};
\ No newline at end of file
+  html: `{% for option in checkboxSelector.options %}<label for="{{checkboxSelector.id}}--{{option.index}}"><input type="{{checkboxSelector.type}}" name="{{checkboxSelector.name}}" id="{{checkboxSelector.id}}--{{option.index}}" value="{{option.value}}" {% if option.value == checkboxSelector.value %}checked{% endif %}>{{option.label}}</label>{% endfor %}`,
```

Comment:
@Snugug 

I understand the _how_ of what you're asking. That is not the issue.

The issue is that this use-case, adding a `fieldset` around a set of checkboxes when there is only one input in an input plugin, will really only apply to _one_ plugin - a plugin which we own. 

I'm sure another developer _may_ create a new input plugin that has only one input that uses checkboxes - but if they did, we should assume the need to create that plugin was beyond what the base input plugin for a single checkbox input could do, _and_ their use-case would call for adding a bunch of other bits of html in the `html` string. Now all those bits will be inside the fieldset too, and those bits may not belong in that fieldset at all.

Also, we would have established that when you create an input plugin that will have multiple checkboxes, there is no need to add a `fieldset` around those checkboxes. So, that would mean that for a developer writing something like [input-plugin-multiple-checkbox](https://github.com/scottnath/content-types/blob/cb470d3592db3c21bc807726dc2d075c49aa9e98/tests/fixtures/input-plugins/input-plugin-multiple-checkbox.js), who wanted to check what the Punchcard creators did with checkboxes, that developer would find a plugin that does not wrap checkboxes and they would (rightfully) assume that the system will wrap fieldsets around all their checkbox sets. Then...to do that in `content-types`, now we will need to search for `{ for option in checkbox.options }` (or any variations on that theme) to add the fieldset around those. 
This is what I meant by saying "I think we'll end up deep in weeds with this"

In short...

* I can easily add this code to `content-types`
* I believe it will set a bad precedent on how to deal with multiple checkboxes from an html standpoint
* I think it would be better to add nunjucks code to deal with adding a fieldset around the checkboxes or not
* fieldsets within fieldsets [look to be legal according to mozilla](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset), so a multi-input plugin for checkboxes would still be html-legal